packages/aws-cdk-lib/aws-apigatewayv2/test/http/api.test.ts

```diff
@@ -234,6 +234,19 @@ describe('HttpApi', () => {
     });
   });
 
+  test('routeSelectionExpression is enabled', () => {
```

Comment:
It might be good to write a test for `RouteSelectionExpression: Match.absent()` when `routeSelectionExpression` is false, as a test for the ternary operator, just in case.